src/test/run-pass/issue-36023.rs

```diff
@@ -0,0 +1,32 @@
+// Copyright 2016 The Rust Project Developers. See the COPYRIGHT
```

Comment:
Travis doesn't use the patched LLVM, so the test *can't* pass on it. See #36138.